### PR TITLE
Change color of attachments button.

### PR DESCRIPTION
**Pull request by edumoreira1506**
This PR changes the color of attachments button, from green to blue. I added too one dialog when one attachment is added but didn't save. Preview: 
Peek 2020-01-20 17-09

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -485,3 +485,5 @@ en:
         default_error: 'Something went wrong, please try again'
       cancel:
         default_cancel: 'Operation canceled'
+    attachments:
+      needs_to_save: "You need to save the story for delete the attachment"
```
Comment (talyssonoc) [inline config/locales/en.yml]:
```suggestion
      needs_to_save: "You need to save the story to delete the attachment"
```

**Comment by edumoreira1506** [comment]
Style updated: 
Peek 2020-01-21 10-32

File: app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js [hunk 2]
```diff
@@ -1,108 +1,99 @@
...
-      return `${className}--drag`
-    }
+    setLoading(true)
+    startLoading();
 
-    return null;
+    upload(newFile)
+      .then(attachment =>
+        onAdd(attachment)
+      )
```
Comment (nolleto) [inline app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js]: You can refactor to:
```js
.then(onAdd)
```

😄

File: app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js [hunk 3]
```diff
@@ -1,108 +1,99 @@
...
+      <AttachmentsList
+        files={story._editing.documents}
+        onDelete={onDelete}
+        disabled={disabled}
+      />
+    </ExpandedStorySection>
+  );
 }
 
 ExpandedStoryAttachments.propTypes = {
```
Comment (nolleto) [inline app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js]: Missing prop type `needsToSave`. 
I believe you set `defaultProps` as `false` and then, update this line of code:
```diff
- needsToSave={story.needsToSave || false}
+ needsToSave={story.needsToSave}
```

File: app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js (hunk 2 above)
Comment (edumoreira1506) [reply app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js]: Nice! Thank you!!

File: app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js (hunk 3 above)
Comment (edumoreira1506) [reply app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js]: Nice, I will update it

**Pull request merged**